docs/tutorials/qpe_with_sampler.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>I am worried about how much this will cost a user to run on the cloud if they have the standard plan....do you think dropping the phase count would help with that? The way it works on cloud today, users would have to sign up for the lite plan to use the simulator.  Might be worth mentioning at the top that this uses the simulator, and folks working on cloud should sign up for the lite plan to use it. Either that or we also show an option that calls a real backend in the code below. I am not sure what would be best so let me know what you think! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Reducing the number of phases would certainly help there. This particular example maps really inefficiently to our topology, so running in real hardware may result in quite deep circuits, so I would advice against providing a real backend call option because the results won't be great.

Reply:
Okay thank you! So it sounds like rather than reducing phase count we should go with the disclaimer that to run this you need the lite Cloud Plan to run with the simulator and/or that running on a real backend with the Standard plan would incur a cost. Maybe @beckykd something similar to what we already have regarding cost?

Reply:
<p>I can add a "Note" or "Warning" info box about to the one used in this how-to guide: https://qiskit.org/documentation/partners/qiskit_ibm_runtime/how_to/specify_the_backend.html (see the bottom).</p>